integrations/database/mysql/mysql.yaml

```diff
@@ -0,0 +1,8 @@
+---
```

Comment:
Should our YAML files have copyright? I see some YAML files in our code have copyright info at the top, others do not.

Reply:
@beerparty , I don't see the copyright in any YAML files.